README.md

```diff
@@ -8,7 +8,7 @@ rclcpp provides the standard C++ API for interacting with ROS 2.
 
 `#include "rclcpp/rclcpp.hpp"` allows use of the most common elements of the ROS 2 system.
 
-Visit the [rclcpp API documentation](http://docs.ros2.org/latest/api/rclcpp/) for a complete list of its main components.
+The link to the latest API documentation can be found on the rclcpp package info page, at the [ROS Index](https://index.ros.org/p/rclcpp/).
```

Comment:
Maybe link https://docs.ros.org/en/rolling/p/rclcpp/ directly?

```suggestion
Visit the [rclcpp API documentation](https://docs.ros.org/en/rolling/p/rclcpp/) for a complete list of its main components.
```

Reply:
@ivanpauno I considered that, but a link to the ROS Index has the added benefit of being instructional to new users (that in general API links for packages can be found through the ROS Index).
But that's up to personal opinion...  Main importance is that it's a correct link, as any new user is likely to search for rclcpp API docs before any other package's API, and google isn't helpful.

Reply:
Yeah, I have to agree with @jrutgeer here that showing people the general place to find API documentation is a good thing, so I would vote to go with the link to index.ros.org.